keps/sig-scheduling/5075-dra-consumable-capacity/README.md

```diff
@@ -122,11 +129,11 @@ and that resource claim has allocated the device.
 With this KEP, independent resource claims (and/or requests within a claim) can allocate shares of the same underlying device.
 This enables resource sharing across pods that are completely unrelated, potentially even across different namespaces.
 
-Additionally, if a device supports sharing, its resource (capacity) can be managed through a defined sharing policy.
+Additionally, if a device supports sharing, its resource (capacity) can be managed through a defined request policy.
 When such a policy is specified, the device may be allocated to multiple independent requests, up to its total capacity,
```

Comment:
this needs to be re-written now that allow multiple allocations and request policy are decoupled.